### ISSUE TITLE
Selective usage of lokinet, via network namespaces or something else

### ISSUE THREAD

**Issue by ple1n** [oxen-io/lokinet#1999]
I haven't found an easy way of selective routing. Lokinet routes all the traffic through it which is unnecessary and exaggerated for daily use, while `.loki` sites can always be accessed without extra configuration/changing the routes.

Maybe lokinet should automatically configure a network namespace, if that is a good choice.

**Comment by majestrate** [comment]
i totally agree and this has been on my mental todo list for almost 3 years.

i think we can do multiple .loki addresses on one lokinet where you have one per user-ns. the user-ns would have a fully isolated net interface and dns server. this would be running in its own seperate process, this is where it gets complex.

the pending open questions:

* how would it communicate to the main lokinet process?
* could we architecture it in a more generic way? e.g. be able to run over the network as well 
* if it can be used over a network would it be over l2 and/or l3? does this conflict with the way we could be doing our poorman's anycast? could it be made the same as the poorman's anycast so we can 2 birds 1 stone it?
